packages/transformers/js/src/helpers.js

```diff
@@ -0,0 +1,37 @@
+export function interopDefault(a) {
```

Comment:
maybe this file could be called `esmodule-helpers.js` or something more specific?